src/utils.jl

```diff
@@ -400,3 +399,16 @@ function record_erupt_end(
     push!(erupt_saved.volume, volume)
     return nothing
 end
+
+function print_timer_log(io::IO, to::TimerOutput)::Nothing
+    print_timer(io, to; compact=true, allocations=false, sortby=:firstexec)
+    return nothing
+end
+
+function vector_uniq(x::Union{Float64,Vector{Float64}})::Vector{Float64}
+    if typeof(x) <: Vector
+        return unique!(x)
+    else
+        return Vector([x])
+    end
+end
```

Comment:
If you end up using a base function, this won't be needed.

Reply:
removed it!